slp_visio/slp_visio/load/objects/diagram_objects.py

```diff
@@ -31,12 +31,11 @@ def get_component_category(self):
     def __str__(self) -> str:
         return '{id: ' + str(self.id) + ', ' \
                + 'name: ' + self.name + ', ' \
-               + 'parent_id: ' + self.name + '}'
+               + 'parent_id: ' + str(self.parent.id if self.parent else None) + '}'
 
     def __repr__(self) -> str:
-        return '{id: ' + str(self.id) + ', ' \
-               + 'name: ' + self.name + ', ' \
-               + 'parent_id: ' + self.name + '}'
+        return self.__str__()
```

Comment:
Before this PR the str() and repr() methods did the same. Both was a copy. I just unified them to avoid duplicity.
I can understand that maybe each one should do different things, but the behavior of them wasn't changed.

The only behavior change I did was fixing the wrong parent_id value

Whatever you want. If you consider better having duplicated with the same behavior, I can change it.

Reply:
Yes, you are absolutely right, this does not really change from a functional point of view. But based on what I found about these two methods, I would say we should be consistent with it unless there is a strong reason to do it this way.

Reply:
What about simply do not override the `repr` method? Makes sense to override the `str` because of simplifying debugging, but I don't think we have a strong reason to change the default behavior of the `repr`.

Reply:
I leave the repr for debugging and development purposes and I restore the repr body